packages/opencensus-core/src/trace/model/no-record/no-record-span-base.ts

```diff
@@ -44,19 +44,19 @@ export abstract class NoRecordSpanBase implements types.Span {
   /** Pointers from the current span to another span */
   links: types.Link[] = [];
   /** If the parent span is in another process. */
-  remoteParent: boolean;
+  remoteParent = false;
   /** The span ID of this span's parent. If it's a root span, must be empty */
-  parentSpanId: string = null;
+  parentSpanId!: string;
   /** The resource name of the span */
-  name: string = null;
+  name!: string;
```

Comment:
Is there a default `name` we give spans? Could we specify it here or even call it `'no-record'` or similar?